src/ngx_http_lua_socket_udp.c

```diff
@@ -54,7 +54,7 @@ static void ngx_http_lua_socket_udp_read_handler(ngx_http_request_t *r,
     ngx_http_lua_socket_udp_upstream_t *u);
 static void ngx_http_lua_socket_udp_handle_success(ngx_http_request_t *r,
     ngx_http_lua_socket_udp_upstream_t *u);
-static ngx_int_t ngx_http_lua_udp_connect(ngx_udp_connection_t *uc);
+static ngx_int_t ngx_http_lua_udp_connect(ngx_resolver_connection_t *uc);
```

Comment:
A closer look at the definition of `ngx_resolver_connection_t` reveals that this type does not serve our purpose well since it contains unused fields like `read_buf` and `write_buf`. Maybe it's time to roll out our own `ngx_http_lua_udp_connection_t` type instead?